alpha/alpha-spring-cloud-starter-consul/src/main/java/org/apache/servicecomb/pack/alpha/server/discovery/consul/AlphaConsulAutoConfiguration.java

```diff
@@ -114,7 +114,11 @@ public void listenInstanceRegisteredEvent(InstanceRegisteredEvent instanceRegist
             NewService newservice =  new NewService();
             newservice.setName(service.getServiceName());
             newservice.setId(service.getServiceId());
-            List<String> tags = consulDiscoveryProperties.getTags();
+            newservice.setAddress(service.getAddress());
```

Comment:
We don't need to update the IP address because it doesn't change

Reply:
I think so, the  IP address is no need to change

Reply:
Sorry about i do not describe my develop environment clearly, and I am sorry for submitting the code without doing a full test( only tested in below environment ).

**version details:** 

> spring boot version: 2.1.4.RELEASE
> spring-cloud-starter-consul version: 2.1.1.RELEASE
> consul version: 1.4.4
> alpha server version: 0.4.0

**application.properties**

> server.port=0
> alpha.server.host=0.0.0.0
> alpha.server.port=0
> alpha.server.initialPort=34500

**One more should be attention**
I **use alpha server as a dependency** in my spring boot project

I use [spring-boot-admin-server](https://github.com/codecentric/spring-boot-admin) for service health monitor

As mentioned above, if `newservice.setPort(service.getServicePort());` this line is missing, consul will register service's health check port as 0, and this will cause health check fails